js-example/src/Components/Bindings/HdWallet.js

```diff
@@ -44,11 +47,24 @@ function base16(u8) {
     return b16;
 }
 
+exports.scrambleImpl = function (entropy) {
+    return function (pass) {
+        var pass_buffer = utf8Encoder.encode(pass);
+        var iv = randomBytes(4);
+        try {
+            return window.Module.PaperWallet.scramble (iv, pass_buffer, entropy);
```

Comment:
Please change this to `window.CardanoCrypto.PaperWallet.scramble`

Reply:
Yes, please do.